src/ansys/fluent/core/fluent_connection.py

```diff
@@ -262,6 +265,14 @@ def get_cortex_connection_properties(self):
 
         return fluent_host_pid, cortex_host, cortex_pid, cortex_pwd
 
+    def is_solver_mode(self):
+        """Returns ``True`` if the Fluent session is in solver mode."""
+        return self.scheme_eval.scheme_eval("(cx-solver-mode?)")
+
+    def exit_server(self):
+        """Scheme call to exit the server."""
```

Comment:
Start description with a verb ending in "s" or "es"

Reply:
Thank you. Done.